macOS/DuckDuckGo/NetworkProtection/ExcludedDomains/AddExcludedDomainView.swift

```diff
@@ -89,15 +109,22 @@ struct AddExcludedDomainView: ModalView {
                 TextField("", text: $domain)
                     .focusedOnAppear()
                     .onChange(of: domain) { domain in
-                        guard let url = URL(trimmedAddressBarString: domain) else { return }
-                        isDefaultActionDisabled = domain.trimmingWhitespace().isEmpty || !url.isValid
+                        isDefaultActionDisabled = Self.isInvalidDomain(domain: domain)
```

Comment:
I'm simply modularizing a bit so I can call the same method from the initializer.